lib/convection/model/template/resource/aws_s3_bucket.rb

```diff
@@ -50,6 +50,50 @@ def website_configuration(&block)
             properties['WebsiteConfiguration'].set(config)
           end
 
+          def full_name(terraform_import: false)
```

Comment:
You should grab the physical ID of the resource using `stack.resources[name]` instead of trying to generate it we do like we do in the internal bucket extension which always gets a predictable suffix.